pkg/package.mk

```diff
@@ -0,0 +1,17 @@
+.PHONY: tag push
+default: push
+
+ORG?=linuxkit
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+NETWORK=0
+
+tag: $(DEPS)
```

Comment:
you may as well add `Dockerfile Makefile $(DEPS)`  here, and then you can remove the (inconsistent) use of `Dockerfile` and `Makefile` in the individual files.